### PR TITLE
Hai reveal tweaks

### PR DESCRIPTION
**Pull request by McloughlinGuy**
## Summary
I fixed typos, added a word in a certain place so it made sense, added some quotation marks in a few places, made it so the pincer beasts were mute (they would talk to you if you asked for help) and deleted the human alliance government, since it didn't need to exist anymore now that it's been replaced by just the regular republic ships. There's a weird thing that happens with the text when the pincer beasts blow up where the quotation marks spawn to the right of the text they should contain, but I couldn't find the data where the self destruct mechanism for them was contained, or anything on the predators that apparently exist. Anyways yeah, just a few tweaks.

### CODE REVIEW

**Comment by McloughlinGuy** [comment]
It's clear to me now: I don't know how to remove a government.

**Comment by quyykk** [comment]
All of mentions of the Human Alliance government:

image

You forgot to remove the last two entries that set the reputation for those governments.

**Comment by McloughlinGuy** [comment]
> All mentions of the Human Alliance government:
> 
> image
> 
> You forgot to remove the last two entries that set the reputation for those governments.

Thank you!

File: data/hai/hai reveal 4 middle.txt [hunk 1]
```diff
@@ -1081,7 +1081,7 @@ mission "Hai Reveal [A10] Demanding Satisfaction"
 		payment 100000
 		conversation
 			`Teeneep makes a request on your approach and then goes to prepare. You are subsequently directed to a restricted landing zone near their government complex. When she reappears Teeneep has prepared to meet the Unfettered in a somewhat surprising manner; she's wearing what looks like a fencing outfit. There are two swords on her back and several balanced knives on her belt. She's wearing a headband with a medallion on it that looks like an athletic medal.`
-			`	She pays you for the trip and explains, "These blades are not ceremonial. Bring your gun with you, but it behind when we reach the chambers, or the Unfettered may think you're an assassin. Trust me, we'll be fine.`
+			`	She pays you for the trip and explains, "These blades are not ceremonial. Bring your gun with you, but put it behind when we reach the chambers, or the Unfettered may think you're an assassin. Trust me, we'll be fine.`
```
Comment (MasterOfGrey) [inline data/hai/hai reveal 4 middle.txt:1084]: This should probably just match what you do later.
```suggestion
			`	She pays you for the trip and explains, "These blades are not ceremonial. Bring your gun with you, but hand it to them when we reach the chambers, or the Unfettered may think you're an assassin. Trust me, we'll be fine.`
```

**Review by MasterOfGrey — CHANGES_REQUESTED** [review]
Thankyou so much for doing this. I hadn't cleared it out only because I hadn't had the confidence in amongst all the other changes at the time to be sure I would safely remove it and not break anything. You've saved me a good bit of time.

Re the tweaks, just a couple small adjustments, but otherwise this is fantastic! :)

**Comment by McloughlinGuy** [comment]
> Thankyou so much for doing this. I hadn't cleared it out only because I hadn't had the confidence in amongst all the other changes at the time to be sure I would safely remove it and not break anything. You've saved me a good bit of time.
> 
> Re the tweaks, just a couple small adjustments, but otherwise this is fantastic! :)

Your welcome! Thanks to quyykk for helping me successfully remove the human alliance too. Glad I could help :)

**Review by roadrunner56 — APPROVED** [review]

**Review by MasterOfGrey — APPROVED** [review]

**Pull request merged**